### PR TITLE
CE-3769: Tests for Community Page Sales Pitch Dialog

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Tests for Community Page Sales Pitch Dialog:
- verify that dialog is shown on 4th pageview
- verify that visiting edit page is not counted as pageview
- verify that clicking on button "Help out" redirect you to Special:Community
- verify that if user has seen dialog (cookie is set) it will not appears again

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SpecialCommunity.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.communitypage;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+public class SpecialCommunity extends WikiBasePageObject {
+
+  public static String COMMUNITY_PAGE_URL = "Special:Community";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SpecialCommunity.java]: MAJOR Make COMMUNITY_PAGE_URL a static final constant or non-public and provide accessors if needed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
MAJOR Make this "public static COMMUNITY_PAGE_URL" field final [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1444)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SalesPitchDialog.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import org.openqa.selenium.support.FindBy;
+
+public class SalesPitchDialog extends WikiBasePageObject {
+
+  private By salesPitchDialog = By.cssSelector("#CommunityPageBenefitsModal");
+
+  @FindBy(css = "#CommunityPageBenefitsModal .community-page-entry-point-button")
+  private WebElement entryPointButton;
+
+  public SalesPitchDialog() { super(); }
```
Comment (Grunny) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SalesPitchDialog.java]: IIRC our coding conventions for Java would requires these to be split over multiple lines.

**Comment by Grunny** [comment]
Minor comment, otherwise LGTM.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SalesPitchDialog.java (hunk 2 above)
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/communitypage/SalesPitchDialog.java]: N o need for the constructor in this situation :) Just delete it

**Pull request merged**